### PR TITLE
36 use webiew api

### PR DESCRIPTION
**Pull request by apupier**
- it doesn't restore the view after close of VSCode but as it doesn't start ATlasMap automatically i think it is fine
- added a single test, i think it opens perspective on the way to test in a more stable way but I would keep it in a separate PR as it is mainly rewriting existing tests
- there is some very little glitch in the UI (blue lines might not appear on import, little blank space on each  border) but I think that it is compensated by the fact that it is embedded in VS Code instead of opening an external browser. It will need to be fixed in AtlasMap code.

### CODE REVIEW

File: src/test/command.test.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import * as sinon from "sinon";
 import * as sinonChai from "sinon-chai";
 import * as vscode from "vscode";
 import * as detect from 'detect-port';
+import * as atlasmapWebView from "./../atlasmapWebView";
```
Comment (lhein) [inline src/test/command.test.ts]: ../atlasmapWebView will be enough. I would also recommend to camel case it correctly. atlasMapWebView

**Comment by apupier** [comment]
> * there is some very little glitch in the UI (blue lines might not appear on import, little blank space on each  border) but I think that it is compensated by the fact that it is embedded in VS Code instead of opening an external browser. It will need to be fixed in AtlasMap code.

or maybe we should wait, I started to play with it and spotted:
-  Remove instance or schema file is not working  https://github.com/atlasmap/atlasmap/issues/752
- blank column on each side https://github.com/atlasmap/atlasmap/issues/749
- import of an ADM does not show mappings (will show them on restart) https://github.com/atlasmap/atlasmap/issues/748

maybe worth someone from QE used to test AtlasMap standalone should do a tour so that we have a better sight on what is working and what is not. it will help to evaluate if the added value to have it inside VS Code compensates the current limitations.
@lhein @paoloantinori what do you think?

**Comment by lhein** [comment]
I would then keep the PR open. QE and AtlasMap team can use your branch to test.

**Comment by apupier** [comment]
I forgot to update the readme, i added a new commit fo rthat.

**Comment by lhein** [comment]
we can close this one, right?

**Comment by apupier** [comment]
yes

**Pull request closed without merging**